projects.py

```diff
@@ -0,0 +1,136 @@
+import argparse
+import sqlite3
+import sys
+import warnings
+
+import pandas.io.sql as pdsql
+import pymysql as mdb
+
+from tt import *
+
+warnings.filterwarnings('ignore')
+
+
+def table_check(args_dict, db):
+    check_table = db.cursor()
+    if args_dict['dbengine'] == 'mysql':
+        check_table.execute('''
+            select count(*) from information_schema.tables where
+                table_name = "{table}";
+        '''.format(table=args_dict['table']))
+    else:
+        check_table.execute('''
+            select count(*) from sqlite_master where type = 'table' and
+                name = "{table}";
+        '''.format(table=args_dict['table']))
+    if check_table.fetchone()[0] == 1:
+        check_table.close()
+        pass
+    else:
+        sys.exit('STOP! The table {} does not exist.'.format(args_dict['table']))
+
+
+def create_projects(args_dict):
+    if args_dict['dbengine']=='mysql':
+        sql_commands = [('''
+            use {db};
+            create table if not exists {table} (
+                id int not null auto_increment,
+                project_name varchar(128),
+                project_code varchar(8),
+                project_bill_code varchar(10),
+                primary key (id)
+            );
+        '''.format(db=args_dict['db'], table=args_dict['table']))]
+    elif args_dict['dbengine'] == 'sqlite':
+        sql_commands = [('''
+            create table if not exists {table} (
+                id integer primary key autoincrement,
+                project_name varchar(128),
+                project_code varchar(8),
+                project_bill_code varchar(10)
+            );
+        '''.format(table=args_dict['table']))]
+    else:
+        sys.exit(DB_ERROR_MESSAGE)
+
+    if args_dict['dbengine'] == 'mysql':
+        sql_commands.insert(0, 'create database if not exists {};'.format(args_dict['db']))
+
+    sql_commands.append(('''
+        insert into {table} (project_name, project_code, project_bill_code)
+            values ('MyFirstProject', 'proj1', '111111');
+    '''.format(table=args_dict['table'])))
+
+    return sql_commands
+
+
+def add_projects(args_dict, db):
+    table_check(args_dict, db)
+    sql_commands = [('''
+        insert into {table} (project_name, project_code, project_bill_code)
+            values ("{desc}", "{code}", "{nbr}");
+    '''.format(table=args_dict['table'],
```

Comment:
it's unclear to me what the idea behind the schema is. `project_bill_code` is straight forward but what is the distinction between `project_name` and `project_code`?